src/stylesheets/global.css

```diff
@@ -1,13 +1,71 @@
 /* Global settings and variables */
 
 @import '~normalize.css';
-@import url('https://fonts.googleapis.com/css?family=Lato:300i,300,400,700');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
 
 :root {
   /* Colors */
-  --color-brand: #0025e4;
+  /* Interface Colors */
   --color-white: #fff;
+  --color-gray-300: #f9f9f9;
+  --color-gray-400: #e0e0e0;
+  --color-gray-500: #9e9e9e;
+  --color-gray-600: #888;
+  --color-gray-700: #666;
+  --color-gray-800: #484848;
+  --color-gray-900: #212121;
+  --color-black: #202020;
 
-  /* Fonts */
-  --font-family-base: 'Lato', 'Arial', sans-serif;
+  /* Message Colors */
+  --color-alert: #eb0000;
+  --color-warn: #ffbc00;
+  --color-success: #2bcf59;
+
+  /* Brand Colors */
+  --color-askdarcel-blue: #276ce5;
+  --color-sheltertech-blue: #0025e4;
+  --color-deep-blue: #202953;
+  --color-midnight-blue: #0f1428;
+
+  /* Font Families */
+  --font-family-hero: 'Merriweather', serif;
+  --font-family-headline: 'Merriweather', serif;
+  --font-family-title: 'Merriweather', serif;
+  --font-family-subtitle: 'Lato', sans-serif;
+  --font-family-caption: 'Lato', sans-serif;
+  --font-family-action: 'Lato', sans-serif;
+  --font-family-body: 'Lato', sans-serif;
```

Comment:
I was thinking that we could create global variables that contained the entirety of the value of the [`font`](https://developer.mozilla.org/en-US/docs/Web/CSS/font) shorthand property, which lets you set multiple properties at once. This would make it easier to be consistent about all the properties that should be set together.

Here's a short example of what I was thinking:

```css
  --font-hero: 300 67px/1.3 'Merriweather', serif;
  --font-body-medium: 400 20px/1.3 'Lato', sans-serif;
```

Something like that could be used in an actual component with `font: var(--font-body-medium);`

It might be useful to still have a variable to hold the font family strings, since one might want to override a font family somewhere else, and we'll want to make the whole list of fonts and fallback fonts consistent. A similar thing goes for the font sizes, since there are just a few specific sizes that we should be using. The font weights I think we probably care less about, since there's only a few weights that are possible to use, and it's hard to get it wrong.

Putting that all together, I think I'd suggest something that looks like this:

```css
  --font-family-merriweather: 'Merriweather', 'Georgia', serif;
  --font-family-lato: 'Lato', 'Arial', serif;

  --font-size-hero: 67px;
  --font-size-body-medium: 20px;

  --font-hero: 300 var(--font-size-hero)/1.3 var(--font-family-merriweather);
  --font-body-medium: 400 var(--font-size-body)/1.3 var(--font-family-lato);
```

I'd also be fine with directly inlining the `var(--font-size-hero)` within the `--font-hero` property definition, since it is a bit more difficult to read when you have too many variable to cross reference, and since all these variables are within the same file, near each other, I would not be as concerned with them drifting apart from each other. In other words, I think the following is an example of when it is OK to not be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself):

```css
  --font-family-merriweather: 'Merriweather', 'Georgia', serif;
  --font-family-lato: 'Lato', 'Arial', serif;

  --font-size-hero: 67px;
  --font-size-body-medium: 20px;

  --font-hero: 300 67px/1.3 var(--font-family-merriweather);
  --font-body-medium: 400 20px/1.3 var(--font-family-lato);
```

Reply:
Thanks for the feedback!

In order to make use of media queries for the mobile font sizes, I'll be directly inlining the `var(--font-size-hero)` within the `--font-hero` property definition.